docs/transpile/customizing-transpilation.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>could you add a "Next steps" box to the end like we have for the other docs pages please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Done.